### ISSUE TITLE
label of instances

### ISSUE THREAD

**Issue by ravigupta131** [mahmoodlab/CLAM#195]
Can anybody suggest how we can get label of instances of each slide

**Comment by fedshyvana** [comment]
Hey, sorry for the late follow-up, I do not have a script for this, but i think you can use 
```logits, Y_prob, Y_hat, A, results_dict = model(features, instance_eval=True) ```
and then instance labels will be a field in the result_dict.
